### PR TITLE
Rollout improvements

### PR DESCRIPTION
**Pull request by mroz22**
todo: 
- [x] rework rollout lib
- [x] implement it to connect
- [x] rework implementation in suite
- [x] keep custom binary as a param to connect
- [x] add btc only to onboarding
- [x] add btc only to firmware modal

### CODE REVIEW

File: packages/integration-tests/projects/suite-web/plugins/index.js [hunk 1]
```diff
@@ -81,9 +82,12 @@ module.exports = on => {
             await controller.disconnect();
             return null;
         },
-        startEmu: async () => {
+        startEmu: async version => {
             await controller.connect();
-            const response = await controller.send({ type: 'emulator-start' });
+            const response = await controller.send({
+                type: 'emulator-start',
+                version: version || '2.1.4',
```
Comment (szymonlesisz) [inline packages/integration-tests/projects/suite-web/plugins/index.js]: default version ('2.1.4') should be set in controller, no?

File: packages/rollout/package.json [hunk 2]
```diff
@@ -1,33 +1,43 @@
 {
     "name": "@trezor/rollout",
-    "version": "1.0.0",
+    "version": "1.0.1-0",
```
Comment (szymonlesisz) [inline packages/rollout/package.json]: real version

File: packages/rollout/package.json [hunk 3]
```diff
@@ -1,33 +1,43 @@
...
-        "cross-fetch": "^3.0.4"
+        "cross-fetch": "^3.0.4",
+        "runtypes": "^4.2.0"
     },
     "devDependencies": {
         "jest-mock-random": "^1.0.3",
-        "rollup": "^0.67.4",
-        "rollup-plugin-cleanup": "^3.0.0",
-        "rollup-plugin-typescript2": "^0.21.2"
+        "rollup": "^2.0.6",
```
Comment (mroz22) [inline packages/rollout/package.json]: remove rollup, use just typescript just like in blockchain link

File: packages/integration-tests/projects/suite-web/plugins/index.js (hunk 1 above)
Comment (mroz22) [reply packages/integration-tests/projects/suite-web/plugins/index.js]: done

File: packages/rollout/package.json (hunk 2 above)
Comment (mroz22) [reply packages/rollout/package.json]: done

File: packages/integration-tests/projects/suite-web/tests/onboarding/unexpected-states.test.ts [hunk 4]
```diff
@@ -38,7 +38,7 @@ describe('Onboarding unexpected states', () => {
     });
 
     it('in case it is not initialzed, just has some firmware on it, use-it-anyway button should appear', () => {
-        cy.task('startEmu');
+        cy.task('startEmu')
```
Comment (szymonlesisz) [inline packages/integration-tests/projects/suite-web/tests/onboarding/unexpected-states.test.ts]: missing comma ?

File: packages/suite/src/views/settings/device/index.tsx [hunk 5]
```diff
@@ -166,22 +166,21 @@ const Settings = ({ device, locks, applySettings, changePin, openModal, goto }:
...
-                            <Translation
-                                id="TR_YOUR_CURRENT_FIRMWARE"
-                                values={{ version: getFwVersion(device) }}
-                            />
+                            <>
+                                <Translation
+                                    id="TR_YOUR_CURRENT_FIRMWARE"
+                                    values={{ version: getFwVersion(device) }}
+                                />
+                                {device.isBitcoinOnly && ' (bitcoin-only)'}
```
Comment (szymonlesisz) [inline packages/suite/src/views/settings/device/index.tsx]: as we decided, do not put `isBitcoinOnly` to the Device.
Make suite utility out of it

**Pull request merged**